packages/nextjs/hooks/scaffold-eth/useEthPrice.ts

```diff
@@ -15,10 +15,9 @@ export const useEthPrice = () => {
     const fetchPriceFromUniswap = async () => {
```

Comment:
since we are using same `fetchPriceFromUniswap` twice, lets move it to `uitls/scaffold-eth` with name `fetchETHPriceFromUniswap` which returns the type of number (ethPrice) and takes in `provider` as argument